### DISCUSSION TITLE
安卓10始终显示任务栏

### DISCUSSION THREAD

**Discussion by HTZG0** [orz12/pilipala#43] [Q&A]
在观看视频全屏的情况下它也会显示，效果如图，（可能是刷机包的问题）![IMG_20240310_160417.jpg](https://github.com/orz12/pilipala/assets/103802635/8d22b4b7-f9cd-499a-aa24-fddc13d51421)

![Screenshot_20240310-160516.jpg](https://github.com/orz12/pilipala/assets/103802635/f08d79c0-65b5-4ec3-8c6b-713e680327ee)
谁能告诉我怎么关掉

**Comment by VillagerTom** [comment]
最新版是否还有此问题？

**Reply by HTZG0** [reply]
不到，我那台手机已经坏了
